### DISCUSSION TITLE
Exclusion grounds at lot level

### DISCUSSION THREAD

**Discussion by alexmohora** [OP-TED/eForms-SDK#704] [Q&A] [answered]
Hello, 

It seems that we have fields for exclusion grounds only at procedure level, BT-67(a)-Procedure and BT-67(b)-Procedure at /*/cac:TenderingTerms/cac:TendererQualificationRequest/cac:SpecificTendererRequirement/cbc:TendererRequirementTypeCode[@listName='exclusion-ground']

But we need to express some of the exclusion grounds, more specific category C from ESPD at LOT level. I have made an example that is validated OK by the CVS. But I didn't find any fields for this, something like  BT-67(a)-Lot and BT-67(b)-Lot in Fields.json

It will be valid for production such an approach?

Please see the form and the response.

Thank you
Alex Mohora

![Exclusion_Grounds_Lot](https://user-images.githubusercontent.com/115160065/229095599-c755d51e-fb30-40f0-99aa-67d78e0e7915.png)

[Exemple_Exclusion_Grounds_Lot.zip](https://github.com/OP-TED/eForms-SDK/files/11120826/Exemple_Exclusion_Grounds_Lot.zip)

**Comment by YvesJo** [comment, marked as answer]
Hi,
Exclusion grounds apply to the whole procedure.
If there are some specific conditions for some lots, you may need to check whether these are exclusion grounds and not award criteria. In case these are still exclusion grounds, you may consider having the specific lots in a separate notice/procedure.
Regarding CVS, rules to prevent unexpected fields aren't yet in place and it the use of such elements is neither expected, nor recommended.
KR
